README.md

```diff
@@ -0,0 +1,112 @@
+# Mesh Connect iOS SDK
+
+iOS library for integrating with Mesh Connect.
+
+## Installation
+
+Add package [LinkSDK](https://github.com/FrontFin/mesh-ios-sdk) in your project's Package Dependencies
+or download [LinkSDK.xcframework](https://github.com/FrontFin/mesh-ios-sdk/LinkSDK.xcframework).
+
+## Get Link token
+
+Link token should be obtained from the POST `/api/v1/linktoken` endpoint. API reference for this request is available [here](https://docs.meshconnect.com/reference/post_api-v1-linktoken). The request must be performed from the server side because it requires the client's secret. You will get the response in the following format:
+```json
+{
+    "content": {
+        "linkToken": "{linkToken}"
+    },
+    "status": "ok",
+    "message": ""
+}
+```
+
+## Launch Link
+
+Create a `LinkConfiguration` instance with the `linkToken` and the callbacks:
+
+```swift
+    let configuration = LinkConfiguration(
+        linkToken: linkToken,
+        settings: LinkSettings?,
+        onIntegrationConnected: onIntegrationConnected,
+        onTransferFinished: onTransferFinished,
+        onEvent: onEvent,
+        onExit: onExit)
+```
+
+The parameter `useSecureOnDeviceStorag`  can be used to enable connected accounts storage in the keychain (disabled by default):
+
+```swift
+    let settings = LinkSettings(useSecureOnDeviceStorage: true)
+```
+
+`LinkStore` class is responsible for adding, removing, and retrieving connected accounts.
+
+The callback `onIntegrationConnected` is called with `LinkPayload` once an integration has been connected.
+It can be either `accessToken(AccessTokenPayload)` or `delayedAuth(DelayedAuthPayload)`
+
+The callback `onTransferFinished` callback is called once a crypto transfer has been executed or failed. The parameter is either `success(TransferFinishedSuccessPayload)` or `error(TransferFinishedErrorPayload)`
+
+The callback `onEvent` is called to provide more details on the user's progress while interacting with the Link.
+This is a list of possible event types, some of them may have additional parameters:
+- `loaded`
+- `integrationConnectionError`
+- `integrationSelected`
+- `credentialsEntered`
+- `transferStarted`
+- `transferPreviewed`
+- `transferPreviewError`
+- `transferExecutionError`
+
+The callback `onExit` is called once a user exits the Link flow, it might be used to dismiss the Link view controller in case the app manages its life cycle (see `LinkHandler.create()`)
```

Comment:
```suggestion
The callback `onExit` is called once a user exits the Link flow. It might be used to dismiss the Link view controller in case the app manages its life cycle (see `LinkHandler.create()`)
```